cel/env.go

```diff
@@ -639,6 +641,14 @@ func getStdEnv() (*Env, error) {
 	return stdEnv, stdEnvErr
 }
 
+// ConfigureLocaleOption configures the Env with the given locale tag.
+func ConfigureLocaleOption(locale string) EnvOption {
```

Comment:
Put this in options.go? Name it `Locale`? (the rest of the options don't have a 'configure' prefix or 'option' suffix.

Reply:
Sorry (wasn't aware about `options.go`, should've checked) - fixed